Description: Do not build unused files
 Skip building binaries that will not be installed.
 The sole purpose of this is to decrease build time.
Forwarded: not-needed
Last-Update: 2020-07-25

--- vfu-4.20.orig/makefile
+++ vfu-4.20/makefile
@@ -28,7 +28,7 @@ AR = ar rv
 CC = gcc
 LD = gcc
 MKDIR = mkdir -p
-MODULES = yascreen vstring vslib vfu
+MODULES = vstring vslib vfu
 RANLIB = ranlib
 RMDIR = rm -rf
 RMFILE = rm -f
@@ -38,7 +38,6 @@ SRC = *.c *.cpp *.cc *.cxx
 ### MODULES ####################################################################
 
 modules:
-	$(MAKE) -C yascreen 
 	$(MAKE) -C vstring 
 	$(MAKE) -C vslib 
 	$(MAKE) -C vfu 
@@ -50,13 +49,11 @@ clean-modules:
 	$(MAKE) -C vfu clean
 
 rebuild-modules:
-	$(MAKE) -C yascreen rebuild
 	$(MAKE) -C vstring rebuild
 	$(MAKE) -C vslib rebuild
 	$(MAKE) -C vfu rebuild
 
 link-modules:
-	$(MAKE) -C yascreen link
 	$(MAKE) -C vstring link
 	$(MAKE) -C vslib link
 	$(MAKE) -C vfu link
--- vfu-4.20.orig/vfu/makefile
+++ vfu-4.20/vfu/makefile
@@ -13,13 +13,13 @@ re: mm_update rebuild
 
 li: mm_update link
 
-all: mm_update vfu vfu.yas vfu-debug vfu.yas-debug 
+all: mm_update vfu
 
 clean: mm_update clean-vfu clean-vfu.yas clean-vfu-debug clean-vfu.yas-debug 
 
-rebuild: mm_update rebuild-vfu rebuild-vfu.yas rebuild-vfu-debug rebuild-vfu.yas-debug 
+rebuild: mm_update rebuild-vfu
 
-link: mm_update link-vfu link-vfu.yas link-vfu-debug link-vfu.yas-debug 
+link: mm_update link-vfu
 
 ### GLOBAL (AND USER) DEFS #####################################################
 
--- vfu-4.20.orig/vstring/makefile
+++ vfu-4.20/vstring/makefile
@@ -13,13 +13,13 @@ re: mm_update rebuild
 
 li: mm_update link
 
-all: mm_update modules vstring.a test 
+all: mm_update vstring.a test 
 
-clean: mm_update clean-modules clean-vstring.a clean-test 
+clean: mm_update clean-vstring.a clean-test 
 
-rebuild: mm_update rebuild-modules rebuild-vstring.a rebuild-test 
+rebuild: mm_update rebuild-vstring.a rebuild-test 
 
-link: mm_update link-modules link-vstring.a link-test 
+link: mm_update link-vstring.a link-test 
 
 ### GLOBAL (AND USER) DEFS #####################################################
 
